web/index.php

```diff
@@ -38,6 +37,8 @@
     }
 }
 
+require __DIR__.'/../app/trusted_proxies.php';
```

Comment:
Why moving require here ?

Reply:
Like this the require is behind the `Debug::enable()` statement in dev and you can see any errors from it.